client/src/utils/helpers.tsx

```diff
@@ -23,3 +23,13 @@ export function isEmail(str: string): boolean {
     /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
   return emailRegex.test(str);
 }
+
+export const STAR_RATING_NAMES = [
```

Comment:
Okay cool I see where you put the allergy option ratings now.

Reply:
Yep so all you have to do to change, add or remove these is change it here in this array, super easy!